stock_picking_invoicing/README.rst

```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Yes, please. i18n files is because Transifex continues writing these files. I'm just going to remove now the source files there.